requirements.txt

```diff
@@ -7,3 +7,4 @@ pymemcache
 cloudpickle
 tqdm
 matplotlib
+datasets
```

Comment:
Since this package is only used in the notebooks, I think it makes more sense to just add it to the `notebooks` tox environment instead.